CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Fix `TypeError` in `Sentry\Monolog\Handler` when the extra data array has numeric keys (#833).
+- Fix parameter and return type of HubInterface (#849)
```

Comment:
I would like to be more specific in regards to what has been done here since it's a really small change, so a phrase like ``Changed the typehints for both the parameter and the return value of the `HubInterface::getCurrentHub` and `HubInterface::setCurrentHub()` methods`` makes more sense.

Reply:
Done! Thx!